internal/slacklog/generator.go

```diff
@@ -282,6 +285,26 @@ func (g *HTMLGenerator) generateMessageDir(channel Channel, key MessageMonthKey,
 			"thumbImageHeight": ThumbImageHeight,
 			"thumbVideoPath":   ThumbVideoPath,
 			"stringsJoin":      strings.Join,
+			"genAttachedURL": func(ts json.Number, fromURL string) string {
+				s, _ := strconv.Atoi(strings.Split(string(ts), ".")[0])
+				unixTime := time.Unix(int64(s), 0)
+				c := strings.Split(fromURL, "/")
+				if len(c) < 2 {
+					return ""
+				}
+				channel := c[len(c) - 2]
+				return fmt.Sprintf("/%s/%d/%02d/#ts-%s", channel, unixTime.Year(), unixTime.Month(), ts)
+			},
+			"isSlackMessage": func(fromURL string) bool {
+				if (len(fromURL) < 25) {
+					return false
+				}
+				if (fromURL[0:25] == "https://vim-jp.slack.com/") {
+					return true
+				} else {
+					return false
+				}
```

Comment:
そのまま return するほうが個人的にはわかりやすくて好みだけど、どちらが Go ぽいかはわからぬ…。
(好みの範疇だと思うのでどちらでもよいです)

```suggestion
				return len(fromURL) >= 25 && fromURL[0:25] == "https://vim-jp.slack.com/"
```